src/DipoleAlgorithms.cc

```diff
@@ -137,34 +137,54 @@ void NaiveDipoleCentroid::measure(
 ) const {
     afw::detection::PeakCatalog const& peaks = source.getFootprint()->getPeaks();
 
-    naiveCentroid(source, exposure, peaks[0].getI(), (peaks[0].getPeakValue() >= 0 ?
-                                                       getPositiveKeys() :
-                                                       getNegativeKeys()));
-    if (peaks.size() > 1) {
-        naiveCentroid(source, exposure, peaks[1].getI(), (peaks[1].getPeakValue() >= 0 ?
-                                                           getPositiveKeys() :
-                                                           getNegativeKeys()));
+    /*
```

Comment:
This block of commented out code should be removed. If you want to keep the comemnted out code, maybe make a copy in a user branch?